web-animations/timing-model/animations/reversed-active-internal-boundaries.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Reversed animation active interval bounds</title>
+<link rel="help" ­href="???">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../../testcommon.js"></script>
+<body>
+<script>
+'use strict'
+
+// TODO(alancutter): Check with birtles whether this should be a WPT given the spec doesn't seem to define this behaviour.
+test(function(t) {
+  var animation = createDiv(t).animate(null, 1);
+  animation.playbackRate = -1;
+
+  animation.currentTime = 0;
+  assert_equals(animation.effect.getComputedTiming().progress, null,
+      'Reversed animation is not in effect when current time is 0');
+
+  animation.currentTime = 1;
+  assert_equals(animation.effect.getComputedTiming().progress, 1,
+      'Reversed animation is in effect when current time is end time');
+
+}, "Reversed animations flip the inclusive/exclusive boundaries of their effect's active interval");
+
+</script>
+</body>
```

Comment:
If I interpret this correctly, github is complaining about a lack of newline at the end of this file.